dashboard/test/controllers/api/v1/projects/section_projects_controller_test.rb

```diff
@@ -26,11 +26,20 @@ class Api::V1::Projects::SectionProjectsControllerTest < ActionController::TestC
     }.to_json
     hidden_project = {id: 33, value: hidden_project_value}
 
+    other_student_project_value = {
+      name: 'Bobs Other App',
+      # project level shouldn't look like this but sometimes does.
+      level: 'https://studio.code.org/projects/weblab',
```

Comment:
maybe a TODO here that this test can come out when we fix the root cause of the long urls?